lib/modules/manager/gleam/artifacts.spec.ts

```diff
@@ -0,0 +1,118 @@
+import _fs from 'fs-extra';
+import { mocked } from '../../../../test/util';
+import { GlobalConfig } from '../../../config/global';
+import type { RepoGlobalConfig } from '../../../config/types';
+import { TEMPORARY_ERROR } from '../../../constants/error-messages';
+import { exec as _exec } from '../../../util/exec';
+import { ExecError } from '../../../util/exec/exec-error';
+import type { UpdateArtifact } from '../types';
+import { updateArtifacts } from './artifacts';
+
+jest.mock('../../../util/exec');
+jest.mock('fs-extra');
+
+const exec = mocked(_exec);
+const fs = mocked(_fs);
+
+const globalConfig: RepoGlobalConfig = {
+  localDir: '',
+};
+
+describe('modules/manager/gleam/artifacts', () => {
```

Comment:
this is depressingly large amount of copy paste from bun... but it's good and all relevant!